### ISSUE TITLE
No IE11 Support with 1.4.1 — transpilation missing

### ISSUE THREAD

**Issue by brandonburkett** [NickKaramoff/shareon#19]
Hello,

We use shareon, but also need to support IE11.  1.3.1 worked great in IE11, but after upgrading to 1.4.1, we receive a syntax error due to the use of an error function here:

```
var urlBuilderMap = {facebook:(d) => ...
```

Is it possible to continue having the dist JS be compiled to es5 for IE11 support?  For now, we have to roll back to 1.3.x.

Thank you
Brandon

**Comment by brandonburkett** [comment]
A quick follow up, it looks like 1.4.0 works as well, so this seems specific to 1.4.1... maybe the typescript conversion is not compiling down to es5?

**Comment by NickKaramoff** [comment]
It's the other way round :D With 1.4.1 I moved from TS to JS for easier compilation and I forgot, that it now doesn't transpile without Babel/Bublé 🤦‍♂️ I'll try to fix it asap, but I can't really promise much until Sep 15 since I'm in the middle of my exams right now 🙄 

For now please refer to using 1.4.0, it has the same functionality as 1.4.1:

```html
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/shareon@1.4.0/dist/shareon.min.css">
<script src="https://cdn.jsdelivr.net/npm/shareon@1.4.0/dist/shareon.min.js"></script>
```

**Comment by brandonburkett** [comment]
Got it, thank you!  1.4.0 works fine in IE11 tested with browserstack.

**Comment by NickKaramoff** [comment]
Hey @brandonburkett, I released the fix in [v1.4.4](https://github.com/NickKaramoff/shareon/releases/tag/v1.4.4) 🎉 I however want to make sure, that it really works in IE11. I just realised that I am using SVG encoded in `data:` URIs -- and IE11 doesn't seem to support this. By that I mean that shareon buttons don't have icons in IE11 -- is this desired?

**Comment by brandonburkett** [comment]
@NickKaramoff thank you.  We'll do some testing, but 99% sure (at least with 1.4.0) the icons were working fine in IE11.  If you are using the same way in 1.4.4, it _should_ work, but we can take a look in browser stack next week.
